scripts/fixate-requirements.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import os
+import os.path
+import sys
+from distutils.version import StrictVersion
+
+
+cwd = os.path.abspath(os.getcwd())
```

Comment:
Instead of using cwd, it would be safer to use location where this file is located and work from there.

```python
BASE_DIR = os.path.dirname(os.path.abspath(__file__))
...
```